### DISCUSSION TITLE
I got a weird issue on my dashboard...

### DISCUSSION THREAD

**Discussion by jamiecropley** [vercel/community#2527] [Bug Report]
![screenshot](https://github.com/vercel/community/assets/17186004/5d83e384-67d1-4375-a107-a00ee2d8e929)

It makes no sense, my site is no way over using that much bandwidth, regardless the numbers indicated make no sense?

**Comment by jamiecropley** [comment]
![image](https://github.com/vercel/community/assets/17186004/deb17df1-8ee1-483f-9510-277cb9c3e6a1)

It's increased a lot in 10 hours lol

It makes no sense, should 500GB bandwidth on the trial but this is giving me zero all of a sudden?

**Comment by leerob** [comment]
Hey! This is a visual bug, we'll get this fixed up as soon as possible 🙏

**Reply by uneknown** [reply]
We have the same problem. Btw. Does the Edge Configuration count towards the KV limit?
![Vercel, fully wrong usage stats...](https://github.com/vercel/community/assets/111852433/340677d7-bb7d-4800-a45c-4043d1183662)

**Reply by BrickheadJohnny** [reply]
Our KV Database count is increasing hourly, at least on our billing page (we're currently at 146), that must be a similar issue (I hope so), right?

**Reply by leerob** [reply]
This should be fixed, are you seeing it resolved for you?

**Reply by BrickheadJohnny** [reply]
Yep, seems like! Waited a couple of hours just to make sure. Thank you so much for fixing it!
